### PR TITLE
1.0.0 upgrade

### PR DESCRIPTION
**Pull request by benbeltzer**
### What does this pr do?
- bumps version to 1.0.0
- removes `environment` param
- removes use of authorization code in `onComplete`
- removes `clientId, template, phone, email` params
- added `showCloseModalButton` param, which allows for differentiation between modal with and without close button

### How should this be tested?
- Use [this](https://github.com/berbix/onboarding-sample-integrations/pull/6) new sample app. See README of that app for instructions on how to do so

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -89,56 +94,32 @@ class BerbixVerify extends React.Component {
...
       clientToken,
       showInModal,
+      showCloseModalButton
     } = this.props;
     if (overrideUrl != null) {
       return overrideUrl;
     }
-    const token = clientToken || continuation;
-    const template = templateKey || role;
+    const token = clientToken;
```
Comment (horaceko) [inline src/index.jsx:115]: You can probably just use `clientToken` directly here.

File: src/index.jsx [hunk 2]
```diff
@@ -89,56 +94,32 @@ class BerbixVerify extends React.Component {
...
     var options = ["sdk=BerbixReact-" + SDK_VERSION];
```
Comment (horaceko) [inline src/index.jsx:116]: Let's switch these over to use template strings, for consistency.

**Review by horaceko — COMMENTED** [review]
Generally looks good! Though, did you mean to nuke the `package-lock.json` file?

File: src/index.jsx (hunk 1 above)
Comment (benbeltzer) [reply src/index.jsx:115]: lollll i wrote the same comment on your PR!

File: src/index.jsx (hunk 2 above)
Comment (benbeltzer) [reply src/index.jsx:116]: agreed

**Comment by benbeltzer** [comment]
@horaceko addressed your feedback!

re:
> did you mean to nuke the package-lock.json file?

I didn't exactly nuke it. When I reran yarn install, it got rid of this block, which we spoke about during the mini-sprint. Do you remember that?

```
"packages": {
    "": {
```

**Comment by horaceko** [comment]
@benbeltzer Aha, yes, that's right. Looks great then!

**Review by horaceko — APPROVED** [review]

**Review by jnnnthnn — APPROVED** [review]

**Pull request merged**